recipes/modern-cpp-kafka/all/conanfile.py

```diff
@@ -0,0 +1,30 @@
+from conans import ConanFile, tools
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class ModernCppKafkaConan(ConanFile):
+    name = "modern-cpp-kafka"
+    description = "A C++ API for Kafka clients (i.e. KafkaProducer, KafkaConsumer, AdminClient)"
+    license = "Apache License 2.0"
+    topics = ("conan", "modern-cpp-kafka", "kafka", "librdkafka", "kafkaproducer", "kafkaconsumer")
```

Comment:
```suggestion
    topics = ("kafka", "librdkafka", "kafkaproducer", "kafkaconsumer")
```
conan and the project name are not valid topics. Topics are keywords when searching a related feature

Reply:
Fixed in the latest commit.